### PR TITLE
SAK-49403: Enhance Log Messages for Improved Turnitin Error Detection

### PR DESCRIPTION
**Pull request by st-manu**
[SAK-49403](https://sakaiproject.atlassian.net/browse/SAK-49403)

[SAK-49403]: https://sakaiproject.atlassian.net/browse/SAK-49403?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by st-manu** [comment]
@mpellicer @stetsche Could you review this, please? Thank you!

File: content-review/impl/turnitin/src/main/java/org/sakaiproject/contentreview/turnitin/TurnitinReviewServiceImpl.java [hunk 1]
```diff
@@ -2280,13 +2280,13 @@ public Map<String, String> getInstructorInfo(String siteId, boolean ignoreUseSou
 					}
 				}
 			} catch (IdUnusedException e) {
-				log.error("Unable to fetch site in getAbsoluteInstructorInfo: " + siteId, e);
+				log.error("Unable to fetch site in {}", siteId);
```
Comment (stetsche) [inline content-review/impl/turnitin/src/main/java/org/sakaiproject/contentreview/turnitin/TurnitinReviewServiceImpl.java:2283]:
```suggestion
				log.error("Unable to fetch site with id {}", siteId);
```

File: content-review/impl/turnitin/src/main/java/org/sakaiproject/contentreview/turnitin/TurnitinReviewServiceImpl.java [hunk 2]
```diff
@@ -2280,13 +2280,13 @@ public Map<String, String> getInstructorInfo(String siteId, boolean ignoreUseSou
...
 			} catch (UserNotDefinedException e) {
-				log.error("Unable to fetch user in getAbsoluteInstructorInfo", e);
+				log.error("Unable to fetch user in {}", siteId);
 			}
 
 			if (inst == null) {
-				log.error("Instructor is null in getAbsoluteInstructorInfo");
+				log.error("Instructor is null in {}", siteId);
```
Comment (stetsche) [inline content-review/impl/turnitin/src/main/java/org/sakaiproject/contentreview/turnitin/TurnitinReviewServiceImpl.java:2289]:
```suggestion
				log.error("Instructor is null. siteId: {}", siteId);
```

File: content-review/impl/turnitin/src/main/java/org/sakaiproject/contentreview/turnitin/TurnitinReviewServiceImpl.java [hunk 3]
```diff
@@ -2280,13 +2280,13 @@ public Map<String, String> getInstructorInfo(String siteId, boolean ignoreUseSou
 					}
 				}
 			} catch (IdUnusedException e) {
-				log.error("Unable to fetch site in getAbsoluteInstructorInfo: " + siteId, e);
+				log.error("Unable to fetch site in {}", siteId);
 			} catch (UserNotDefinedException e) {
-				log.error("Unable to fetch user in getAbsoluteInstructorInfo", e);
+				log.error("Unable to fetch user in {}", siteId);
```
Comment (stetsche) [inline content-review/impl/turnitin/src/main/java/org/sakaiproject/contentreview/turnitin/TurnitinReviewServiceImpl.java:2285]:
```suggestion
				log.error("Unable to fetch current user on site with id {}", siteId);
```

**Review by stetsche — CHANGES_REQUESTED** [review]
Good job @st-manu! I just added some minor suggestions to make the log even clearer. Please take a look.

**Comment by mpellicer** [comment]
Great job Manu @st-manu

**Comment by st-manu** [comment]
Thanks @stetsche !!

**Comment by mpellicer** [comment]
CLA can't be confirmed since the link does not work anymore.

http://licensing.apereo.org/completed-clas

**Comment by mpellicer** [comment]
@st-manu Do you remember if you signed the ICLA here?

https://www.apereo.org/licensing/agreements/icla

**Comment by st-manu** [comment]
@mpellicer No, I haven't signed that. I'll sign it now.

**Review by stetsche — APPROVED** [review]

**Comment by mpellicer** [comment]
CLA confirmed.

**Pull request merged**